provider/cmd/pulumi-resource-equinix/schema.json

```diff
@@ -12810,7 +12810,7 @@
    Package = new Equinix.Fabric.Inputs.CloudRouterPackageArgs\n        {\n            Code = \"STANDARD\",\n        },\n        Project = new Equinix.Fabric.Inputs.CloudRouterProjectArgs\n        {\n            ProjectId = \"776847000642406\",\n        },\n        Account = new Equinix.Fabric.Inputs.CloudRouterAccountArgs\n        {\n            AccountNumber = 203612,\n        },\n    });\n\n});\n```\n```java\npackage generated_program;\n\nimport com.pulumi.Context;\nimport com.pulumi.Pulumi;\nimport com.pulumi.core.Output;\nimport com.pulumi.equinix.fabric.CloudRouter;\nimport com.pulumi.equinix.fabric.CloudRouterArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterNotificationArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterOrderArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterLocationArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterPackageArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterProjectArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterAccountArgs;\nimport java.util.List;\nimport java.util.ArrayList;\nimport java.util.Map;\nimport java.io.File;\nimport java.nio.file.Files;\nimport java.nio.file.Paths;\n\npublic class App {\n    public static void main(String[] args) {\n        Pulumi.run(App::stack);\n    }\n\n    public static void stack(Context ctx) {\n        var newCloudRouter = new CloudRouter(\"newCloudRouter\", CloudRouterArgs.builder()\n            .name(\"Router-SV\")\n            .type(\"XF_ROUTER\")\n            .notifications(CloudRouterNotificationArgs.builder()\n                .type(\"ALL\")\n                .emails(                \n                    \"example@equinix.com\",\n                    \"test1@equinix.com\")\n                .build())\n            .order(CloudRouterOrderArgs.builder()\n                .purchaseOrderNumber(\"1-323292\")\n                .build())\n            .location(CloudRouterLocationArgs.builder()\n                .metroCode(\"SV\")\n                .build())\n            .package_(CloudRouterPackageArgs.builder()\n                .code(\"STANDARD\")\n                .build())\n            .project(CloudRouterProjectArgs.builder()\n                .projectId(\"776847000642406\")\n                .build())\n            .account(CloudRouterAccountArgs.builder()\n                .accountNumber(\"203612\")\n                .build())\n            .build());\n\n    }\n}\n```\n```yaml\nresources:\n  newCloudRouter:\n    type: equinix:fabric:CloudRouter\n    name: new_cloud_router\n    properties:\n      name: Router-SV\n      type: XF_ROUTER\n      notifications:\n        - type: ALL\n          emails:\n            - example@equinix.com\n            - test1@equinix.com\n      order:\n        purchaseOrderNumber: 1-323292\n      location:\n        metroCode: SV\n      package:\n        code: STANDARD\n      project:\n        projectId: '776847000642406'\n      account:\n        accountNumber: '203612'\n```\n{{% /example %}}\n\n{{% /examples %}}",
```

Comment:
These are the uncommitted changes that are causing the release job to fail.  I don't know why `make only_tfgen` produces these changes.

The diff is hard to parse visually so I also can't tell what exactly has changed about these descriptions.

Reply:
because of this https://github.com/equinix/pulumi-equinix/pull/152#discussion_r1722135351
#140 those manual changes in the examples. The code of the examples are added to the each description once generated

Reply:
I don't understand why you're referencing #140.  It hasn't been merged so why would the changes in that unmerged PR affect the release job that runs against the `main` branch?

Reply:
oops sorry! I don't know how I could have gotten confused, I thought 140 was already merged